cobra_commander/spec/cobra_commander/cli_spec.rb

```diff
@@ -154,50 +122,53 @@
       run_command_and_stop("cobra tree -a #{umbrella.path}", fail_on_error: true)
 
       expected_output = <<~OUTPUT
-        b
-        └── e
-        e
-        f
-        └── b
-            └── e
-        g
-        ├── e
-        └── f
-            └── b
-                └── e
-        h
-        ├── b
-        │   └── e
-        └── f
-            └── b
-                └── e
-        node_manifest
-        ├── b
-        │   └── e
-        ├── e
-        ├── f
-        │   └── b
-        │       └── e
-        └── g
-            ├── e
-            └── f
-                └── b
-                    └── e
-        a
-        ├── b
-        │   └── e
-        └── c
-            └── b
-                └── e
-        c
-        └── b
-            └── e
-        d
-        ├── b
-        │   └── e
-        └── c
-            └── b
-                └── e
+        auth
+        directory
+        └── auth
+        finance
+        ├── auth
+        ├── directory
+        │   └── auth
+        └── payroll
+            └── directory
+                └── auth
+        hr
+        ├── auth
+        └── directory
+            └── auth
+        payroll
+        └── directory
+            └── auth
+        sales
+        ├── auth
+        ├── directory
+        │   └── auth
+        └── finance
+            ├── auth
+            ├── directory
+            │   └── auth
+            └── payroll
+                └── directory
+                    └── auth
```

Comment:
These file trees look a lot more realistic when are fixtures have names instead of single letters.